docs/plugins/dffml_model.rst

```diff
@@ -154,77 +154,55 @@ Generating train and test data
   make sure to take a BACKUP of files with same name in the directory
   from where this command is run as it overwrites any existing files.
 
-.. code-block:: console
+.. literalinclude:: /../model/tensorflow/examples/tfdnnr/train_data.sh
+
+.. literalinclude:: /../model/tensorflow/examples/tfdnnr/test_data.sh
+
+Train the model
+
+.. literalinclude:: /../model/tensorflow/examples/tfdnnr/train.sh
+
+Assess the accuracy
+
+.. literalinclude:: /../model/tensorflow/examples/tfdnnr/accuracy.sh
+
+Output
+
+.. code-block::
 
-    $ cat > train.csv << EOF
-    Feature1,Feature2,TARGET
-    0.93,0.68,3.89
-    0.24,0.42,1.75
-    0.36,0.68,2.75
-    0.53,0.31,2.00
-    0.29,0.25,1.32
-    0.29,0.52,2.14
-    EOF
-    $ cat > test.csv << EOF
-    Feature1,Feature2,TARGET
-    0.57,0.84,3.65
-    0.95,0.19,2.46
-    0.23,0.15,0.93
-    EOF
-    $ dffml train \
-        -model tfdnnr \
-        -model-epochs 300 \
-        -model-steps 2000 \
-        -model-predict TARGET:float:1 \
-        -model-hidden 8 16 8 \
-        -sources s=csv \
-        -source-filename train.csv \
-        -model-features \
-          Feature1:float:1 \
-          Feature2:float:1 \
-        -log debug
-    Enabling debug log shows tensorflow losses...
-    $ dffml accuracy \
-        -model tfdnnr \
-        -model-predict TARGET:float:1 \
-        -model-hidden 8 16 8 \
-        -sources s=csv \
-        -source-filename test.csv \
-        -model-features \
-          Feature1:float:1 \
-          Feature2:float:1 \
-        -log critical
     0.9468210011
-    $ echo -e 'Feature1,Feature2,TARGET\n0.21,0.18,0.84\n' | \
-      dffml predict all \
-        -model tfdnnr \
-        -model-predict TARGET:float:1 \
-        -model-hidden 8 16 8 \
-        -sources s=csv \
-        -source-filename /dev/stdin \
-        -model-features \
-          Feature1:float:1 \
-          Feature2:float:1 \
-        -log critical
+
+Make a prediction
+
+.. literalinclude:: /../model/tensorflow/examples/tfdnnr/predict.sh
+
+Output
+
+.. code-block:: json
+    
     [
-        {
-            "extra": {},
-            "features": {
-                "Feature1": 0.21,
-                "Feature2": 0.18,
-                "TARGET": 0.84
-            },
-            "last_updated": "2019-10-24T15:26:41Z",
-            "prediction": {
-                "TARGET" : {
-                    "confidence": NaN,
-                    "value": 1.1983429193496704
-                }
-            },
-            "key": 0
-        }
+      {
+        "extra": {},
+        "features": {
+            "Feature1": 0.21,
+            "Feature2": 0.18,
+            "TARGET": 0.84
+        },
+        "last_updated": "2019-10-24T15:26:41Z",
+        "prediction": {
+            "TARGET" : {
+                "confidence": NaN,
+                "value": 1.1983429193496704
+            }
+        },
+        "key": 0
+      }
```

Comment:
It looks like some of this JSON is indented with four spaces and some is indented with two. Can we make it all four?